.github/workflows/benchmarking.yml

```diff
@@ -0,0 +1,19 @@
+name: Run Benchmarks
+
+on: [push, pull_request]
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Run Go Benchmark Action
+        uses: patrickhuie19/benchmark-action@v0.2
```

Comment:
do we have a chainlink repo or some other standard place that this ought to stored?

i read the action in the other repo and have a few questions/comments:
- it's not clear what syntax is needed in the PR body to set the benchmarks. this needs to be in the description. based on my reading of the sed, i think the format is expected to be ... BENCHMARK: A,B,C ... where `BENCHMARK:` is the token and the value is supposed be a CSV
- i don't understand if there is a set of on-by-default benchmarks. the added code here suggests yes, but i didn't find clear default logic in the other repo or here

Reply:
> do we have a chainlink repo or some other standard place that this ought to be stored?

Yep - and I'll port/copy the action I created in my personal repo over to there. I was working in my repo for several reasons: 1. I didn't have write access to what I needed, and 2. Testing this stuff requires a ton of commit churn, so I needed to get commits and tags up ASAP to start hacking on this

> it's not clear what syntax is needed in the PR body to set the benchmarks. this needs to be in the description. based on my reading of the sed, i think the format is expected to be ... BENCHMARK: A,B,C ... where BENCHMARK: is the token and the value is supposed be a CSV

README: https://github.com/patrickhuie19/benchmark-action

> i don't understand if there is a set of on-by-default benchmarks. the added code here suggests yes, but i didn't find clear default logic in the other repo or here

On-by default benchmarks have to be specified as an input to the action so that those benchmarks are run for merges. There are no PR descriptions for merges.